test/cog/template_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Cog.TemplateTest do
+  use ExUnit.Case
+  alias Cog.Template
+
+  test "rendering json for the slack adapter" do
```

Comment:
Mwahaha... testing the templates independent of any command execution is :+1: :+1: :100: :confetti_ball: